examples/simple_example/CMakeLists.txt

```diff
@@ -15,4 +15,10 @@ add_custom_command(TARGET simple_example POST_BUILD
   COMMAND_EXPAND_LISTS
 )
 
-target_link_libraries(simple_example PUBLIC dart_dll)
\ No newline at end of file
+add_dependencies(simple_example ALWAYS_DO_POST_BUILD)
+
+target_link_libraries(simple_example PUBLIC dart_dll)
+
+if (MSVC)
```

Comment:
Do you have to wrap this in MSVC? `VS_DEBUGGER_WORKING_DIRECTORY` should only work for MSVC.

Reply:
No i build with VSCode but you have it on other samples why not in simple ?
There ist also an "Problem" realtime_example is the only sample wich is in the Debug folder all other samples only
in example\<name>\ debug folder. I understand not why.

my practice is put the output too an bin folder i no that makes small count of developer it is easier to wirte launch.json  for debugging

Reply:
Yeah, to be honest I'm not as good with CMake as I'd like, so I'm not surprised there are weird issues.

I'll leave this in if it's working for you and take another pass at the CMake files at some point.